crates/solver-service/AGENTS.md

```
@@ -0,0 +1,37 @@
+
+`src/server.rs` also reads `REDIS_URL` independently for the admin Redis client.
+
+## Run locally
+
+```
+cargo run -p solver-service
+```
+
+Or, since `solver-service` is a default member:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)